SConstruct

```diff
@@ -61,11 +61,14 @@ elif platform_arg == "javascript":
     # Use generic POSIX build toolchain for Emscripten.
     custom_tools = ["cc", "c++", "ar", "link", "textfile", "zip"]
 
-env_base = Environment(tools=custom_tools)
-if "TERM" in os.environ:
-    env_base["ENV"]["TERM"] = os.environ["TERM"]
-env_base.AppendENVPath("PATH", os.getenv("PATH"))
-env_base.AppendENVPath("PKG_CONFIG_PATH", os.getenv("PKG_CONFIG_PATH"))
+env_base = Environment(
+    ENV={
+        "PATH": os.getenv("PATH"),
+        "PKG_CONFIG_PATH": os.getenv("PKG_CONFIG_PATH"),
+        "TERM": os.getenv("TERM"),
```

Comment:
This might change behavior a bit as the previous code only defined it if `"TERM"` was in `os.environ`. Now it will define it to `None` if that's not the case.

Reply:
This was added in 2bf4553fe056c1fab5367dfae62426d3c7cf8168 so it's fairly low risk... Not sure how it's used exactly by SCons, I don't see any reference to `TERM` in the SCons source code.